src/style/addnote.css

```diff
@@ -0,0 +1,111 @@
+.add-note-container {
+  width: auto;
+  height: fit-content;
+  margin: auto;
+  padding: 2rem;
+}
+
+.note-title-container,
+.note-body-container,
+.note-tags-container,
+.note-timer-container {
+  min-width: 20rem;
+  padding-bottom: 1rem;
+}
+
+.note-textarea {
+  padding: 1rem;
+  font-size: 1.1rem;
+  border-radius: 0.5rem;
+}
+
+.note-tags-container,
+.note-timer-container {
+  padding-top: 0.6rem;
+  margin-bottom: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.tags-input {
+  display: flex;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  min-height: 48px;
```

Comment:
Using rems would be better here. 

Throughout code try to consistent with rems or px. Right now it's a mix of both. 

Rest of the code looks well structured. Good work.